dev/NavigationView/TestUI/Hierarchical/HierarchicalNavigationViewDataBinding.xaml.cs

```diff
@@ -71,7 +71,7 @@ public HierarchicalNavigationViewDataBinding()
 
             categories.Add(new Category("Menu Item 1", "MI1", "Icon", categories2, false));
             categories.Add(new Category("Menu Item 6 (Selectable)", "MI6", "Icon", categories4, true));
-            categories.Add(new Category("Menu Item 10", "MI10", "Icon", null, true));
+            categories.Add(new Category("Menu Item 10", "MI10", "Icon", new ObservableCollection<Category>(), true));
```

Comment:
this should follow the same format as others such as `var categories10 = new ObservableCollection<Category>()`

Reply:
Fixed now.